src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -31,10 +31,36 @@ namespace Microsoft.ML.Transforms.Text
     /// </summary>
     public interface IStopWordsRemoverOptions { }
 
-    // A transform that turns a collection of text documents into numerical feature vectors. The feature vectors are counts
-    // of (word or character) ngrams in a given text. It offers ngram hashing (finding the ngram token string name to feature
-    // integer index mapping through hashing) as an option.
-    /// <include file='doc.xml' path='doc/members/member[@name="TextFeaturizingEstimator "]/*' />
+    /// <summary>
+    ///  A transform that turns a collection of text documents into numerical feature vectors.
```

Comment:
>A transform [](start = 9, length = 11)

Would avoid starting with `a transform` as the description of an estimator.

How about starting with `Featurizes text data by ...`